relay-general/src/protocol/device_class.rs

```diff
@@ -17,15 +17,10 @@ impl DeviceClass {
         if let Some(Context::Device(ref device)) = contexts.get_context("device") {
             if let Some(family) = device.family.value() {
                 if family == "iPhone" || family == "iOS" || family == "iOS-Device" {
-                    if let Some(processor_frequency) = device.processor_frequency.value() {
-                        if processor_frequency < &2000 {
-                            return Some(DeviceClass::LOW);
-                        } else if processor_frequency < &3000 {
-                            return Some(DeviceClass::MEDIUM);
-                        } else {
-                            return Some(DeviceClass::HIGH);
-                        }
+                    if let Some(model) = device.model.value() {
```

Comment:
Related to my comment above -- what if there's no model, but the processor frequency is available? We can prioritize using the model over the frequency, but changing this behavior is a breaking change. Shouldn't the tests below still pass, _and_ new tests for this new functionality also pass?

Reply:
Hey @iker-barriocanal , so this feature isn't being used by/released to any customers yet, we are in the process of rolling it out which is why I think we can go back on this logic. I confirmed with the sdk team that iOS devices are unable to send `processor_frequency` so there should never be a case where we would enter the code block to check for `processor_frequency` which is why I removed it to simplify. Let me know if this is fine!